.github/workflows/build_and_test_server.yml

```diff
@@ -18,7 +18,13 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: '^1.19.4' # The Go version to download (if necessary) and use.
+      - name: Install go linter
```

Comment:
How about we add the linter into our makefile and then call it from here?
With the linter in makefile we will have easier time running it during development.

Reply:
Sure, will do.